system-contracts/bootloader/bootloader.yul

```diff
@@ -3322,7 +3314,7 @@ object "Bootloader" {
                 }
 
                 let gasLimitValue := getGasLimit(innerTxDataOffset)
-                if iszero(validateUint32(gasLimitValue)) {
+                if iszero(validateUint64(gasLimitValue)) {
                     assertionError("Encoding gasLimit")
```

Comment:
Maybe we can explicity include that the `gasLimitValue` is above the allowed range of max_uint64 in the error?